apps/admin/src/components/OperationForm/Controls/related-resource-control/index.tsx

```diff
@@ -46,7 +46,7 @@ export const RelatedResourceControl: FC<IRelatedResourceControl> = ({
     operationInfo?.parameters,
     {
       staleTime: 300000, // 5 min
-      enabled: !operationInfo?.pauseOperation && hasAccess,
+      disabled: operationInfo?.pauseOperation && hasAccess,
```

Comment:
should this be operationInfo?.pauseOperation || !hasAccess

Reply:
Ah yes. Good catch!